koku/masu/util/ocp/common.py

```diff
@@ -402,3 +402,14 @@ def get_amortized_monthly_cost_model_rate(monthly_rate, start_date):
 
     days_in_month = dh().days_in_month(start_date)
     return Decimal(monthly_rate) / days_in_month
+
+
+def get_latest_operator_version():
+    """Get s the latest operator version we have released based on OPERATOR_VERSIONS dict"""
+    latest_version_num = None
+    for version in OPERATOR_VERSIONS.values():
+        version_num = version.split(":")[-1]
+        version_num = version_num.lstrip("v")
+        if latest_version_num is None or version_num > latest_version_num:
+            latest_version_num = version_num
+    return latest_version_num
```

Comment:
This version comparison will fail if versions ever gets to double digits in any of the versioning fields since it is comparing strings.

```
>>> "3.1.10" > "3.1.1"
True
>>> "3.1.10" > "3.1.9"
False
>>> "3.1.10" > "3.1.2"
False
>>> "3.10.0" > "3.9.1"
False
>>> "10.1.1" > "9.1.1"
False
```

Might be a good spot to try and use version handling from packaging: https://packaging.pypa.io/en/stable/version.html 
```
>>> from packaging.version import Version
>>> Version("10.1.1") > Version("9.1.1")
True
>>> Version("9.1.0") > Version("9.1.1")
False
```

Reply:
Thats a great idea